doc/content/clocks.md

```diff
@@ -181,7 +181,7 @@ If you want to run an output as fast as your CPU allows,
 just attach it to a new clock without synchronization:
 
 ```liquidsoap
-clock.assign_new(sync=false,[output.file(%vorbis,"audio.ogg",source)])
+clock.assign_new(sync="none",[output.file(%vorbis,"audio.ogg",source)])
```

Comment:
A question reading this again: do we want to used `clock.assign_new` with `sync` being something else than `"none"` (and example would be nice...)? At least shouldn't this be the default value for the argument?

Reply:
No, auto is what it should be as it covers most of the "simple" situations. I'll add some doc, though.